disassembling/esil.md

```diff
@@ -105,6 +105,7 @@ TRAP  | src | Trap | Trap signal |
  **>>** | src,dst | Shift Right | stack = dst >> src | &gt; "ae 1,4,>>"<br>0x2<br>&gt; "ae 2,4,>>"<br>0x1
  **<<<** | src,dst | Rotate Left | stack=dst ROL src | &gt; "ae 31,1,<<<"<br>0x80000000<br>&gt; "ae 32,1,<<<"<br>0x1
 **>>>** | src,dst | Rotate Right | stack=dst ROR src | &gt; "ae 1,1,>>>"<br>0x80000000<br>&gt; "ae 32,1,>>>"<br>0x1
+**>>>>** | src,dst | Arithmetic Shift Right | stack=dst >> src | &gt; "ae -4,r_01,="<br>&gt; "ae 1,r_01,>>>>"<br>&gt;"ar r_01"<br>0xfffffffe
```

Comment:
Wrong

Reply:
What's wrong ? I have tested in radare2 I got the same result.